source/_patterns/04-pages/home.mustache

```diff
@@ -9,49 +9,51 @@
 
     <main role="main" class="main" id="maincontent">
 
-      {{#carousel}}
-        {{> organisms-carousel}}
-      {{/carousel}}
+        {{#hero-banner}}
+        <div class="wrapper">
+            {{> organisms-hero-banner}}
+        </div>
+        {{/hero-banner}}
 
-      {{#leadParas}}
+        {{#leadParas}}
         <div class="wrapper">
-          {{> molecules-lead-paras}}
+            {{> molecules-lead-paras}}
         </div>
-      {{/leadParas}}
+        {{/leadParas}}
 
-      {{#sectionsListingLink}}
-      <div class="wrapper">
-        {{> atoms-section-listing-link}}
-      </div>
-      {{/sectionsListingLink}}
+        {{#sectionsListingLink}}
+        <div class="wrapper">
+            {{> atoms-section-listing-link}}
+        </div>
+        {{/sectionsListingLink}}
 
-      <div class="wrapper wrapper--listing">
-        <div class="grid">
-          <div class="grid__item one-whole x-large--eight-twelfths listing-column-main">
+        <div class="wrapper wrapper--listing">
+            <div class="grid">
+                <div class="grid__item one-whole x-large--eight-twelfths listing-column-main">
 
-            {{#mainTeasers}}
-              {{> organisms-listing-teasers}}
-            {{/mainTeasers}}
+                    {{#mainTeasers}}
+                    {{> organisms-listing-teasers}}
+                    {{/mainTeasers}}
 
-          </div>
+                </div>
 
-          <div class="grid__item one-whole x-large--four-twelfths listing-column-secondary">
+                <div class="grid__item one-whole x-large--four-twelfths listing-column-secondary">
 
-              {{#secondaryTeasers}}
-                {{> organisms-listing-teasers}}
-              {{/secondaryTeasers}}
+                    {{#secondaryTeasers}}
+                    {{> organisms-listing-teasers}}
+                    {{/secondaryTeasers}}
 
-          </div>
+                </div>
 
-          <div>
-      </div>
+                <div>
+                </div>
 
 
-      {{#sectionsListing}}
-        <div class="wrapper u-pad-top-lg">
-          {{> molecules-section-listing}}
-        </div>
-      {{/sectionsListing}}
+                {{#sectionsListing}}
+                <div class="wrapper u-pad-top-lg">
```

Comment:
Change class name so we know what it is attached to

Reply:
The class name is not new and it is not part of the Hero banner logic. I can change it but it will require three more classes changes: .u-pad-top, .u-pad-top-lg, .u-pad-bottom, .u-pad-bottom-lg. Do you want me to change them?

Reply:
@viktoria-mahmud no need to change. @discodavey we can raise as a separate ticket if value in changing. Would be useful to change things in this file unrelated to your changes. (format only)

Reply:
The additional white spaces are removed from this file